src/test/java/com/google/api/codegen/testsrc/library_pkg2.yaml

```diff
@@ -4,8 +4,8 @@ artifact_type: GAPIC
 organization_name: google-cloud
 proto_path: google/library
 proto_deps:
-- google-common-protos
-- google-some-other-package-v1
-proto_test_deps:
-- google-some-test-package-v1
+- name: google-common-protos
```

Comment:
Do we need to make a similar change anywhere else?

Reply:
I also pushed an artman change to send this format: https://github.com/googleapis/artman/pull/444